packages/flutter/lib/src/widgets/animated_scroll_view.dart

```diff
@@ -1094,4 +1128,22 @@ abstract class _SliverAnimatedMultiBoxAdaptorState<T extends _SliverAnimatedMult
       setState(() => _itemsCount -= 1);
     });
   }
+
+  /// Remove all the items and start an animation that will be passed to
+  /// `builder` when the items are visible.
+  ///
+  /// Items are removed immediately. However, the
+  /// items will still appear for `duration` and during that time
+  /// `builder` must construct its widget as needed.
+  ///
+  /// This method's semantics are the same as Dart's [List.clear] method: it
+  /// removes all the items in the list
+  ///
+  void removeAllItems(AnimatedRemovedItemBuilder builder, { Duration duration = _kDuration }) {
+    assert(builder != null);
+    assert(duration != null);
```

Comment:
This is what is causing the analysis failure. These asserts are unnecessary since these are not nullable.